integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkCommonIT.java

```diff
@@ -598,6 +603,26 @@ protected void checkDirExists(Path dir) {
     }
   }
 
+  protected void checkDataFileExists(Path dir) {
```

Comment:
`checkDataFileExists` suggest just check the data file exists under dir and no need to check fileList.size and file content.